fighters/robin/src/status.rs

```diff
@@ -0,0 +1,15 @@
+use super::*;
+use globals::*;
+
+#[status_script(agent = "reflet", status = FIGHTER_STATUS_KIND_ATTACK_AIR, condition = LUA_SCRIPT_STATUS_FUNC_INIT_STATUS)]
+pub unsafe fn init_attack_air(fighter: &mut L2CFighterCommon) -> L2CValue {
+    VisibilityModule::set_int64(fighter.module_accessor, Hash40::new("sword").hash as i64, Hash40::new("sword_normal").hash as i64);
+    WorkModule::off_flag(fighter.module_accessor, *FIGHTER_REFLET_INSTANCE_WORK_ID_FLAG_THUNDER_SWORD_ON);
+    0.into()
```

Comment:
Add `fighter.sub_attack_air_uniq_process_init();` to this, even though aerials dealing decreased shield stun was removed, in the unlikely case that value gets changed Robin's aerials will no longer be consistent with the rest of the cast.